bitrise/bitrise.go

```diff
@@ -40,8 +40,8 @@ type startRequest struct {
 
 // StartResponse ...
 type StartResponse struct {
-	Status            string `json:"message"`
-	Message           string `json:"status"`
+	Status            string `json:"status"`
+	Message           string `json:"message"`
```

Comment:
Could this cause an issue before? Do we even use it?

Reply:
Nope, these were unused fortunately.

Reply:
Should we remove them?

Reply:
I prefer defining the whole API response structure, even though some fields are currently unused. This is how we define APIs in other repos (eg. ios-autoprovision), and I think its helpful when working on a new feature and looking for other stuff exposed by an API.

Reply:
Makes sense, let's keep them!